### ISSUE TITLE
Tabs With Manual Activation example: consistency of activation Behaviour when deleting a tab

### ISSUE THREAD

**Issue by jscholes** [w3c/aria-practices#1740]
The aim of the [Tabs With Manual Activation example](https://w3c.github.io/aria-practices/examples/tabs/tabs-2/tabs.html) is to demonstrate the mark-up and behaviour of a tab system where automatic activation isn't desirable.  The consistency of this behaviour may be undermined during tab deletion; specifically:

1. Open the example page which has the first tab ("Nils Frahm") activated by default.
2. With a screen reader in focus/forms mode, move focus to the third tab ("Joke") without activating it.
3. Press Delete.

At this point, the second tab ("Agnes Obel") becomes automatically activated, even though it wasn't explicitly activated by the user before deletion of the third tab.  In other words, the first tab's content was showing, so there was no explicit need to activate a different tab after deletion of the third one.

My initial thoughts are that:

1. If the third tab is deleted while in an activated state, it is necessary to automatically activate the tab to the left.  But...
2. If the third tab is deleted while one of the other tabs is active, it is of course necessary to move focus to a different tab in the list, but not to unexpectedly change the activated one.

The outstanding question, therefore, would be where focus should move when the third tab is deleted without another tab being automatically activated.  This could be the tab to the left, even if that one isn't activated, or the previously activated tab.  Practically speaking, that would mean:

1. If the first tab is activated and the third gets deleted, focus could land on the first tab or the second tab.
2. If the second tab is active when the third is deleted, focus should probably land on the second one regardless.

**Comment by jongund** [comment]
I agree that deleting the third tab should not change that activated tab, unless the deleted tab was the one with the active state.  I think the example also needs a visual affordance that it can be deleted, for example a box with an X (e.g. ☒).
